packages/fast-components-react-msft/src/slider-label/slider-label.schema.ts

```diff
@@ -6,25 +6,30 @@ export default {
     id: "@microsoft/fast-components-react-msft/slider-label",
     formPluginId: "@microsoft/fast-components-react-base/slider-label",
     properties: {
-        maxValuePositionBinding: {
-            title: "Upper value position binding",
-            type: "string",
-            enum: [
-                "selectedRangeMin",
-                "selectedRangeMax",
-                "totalRangeMin",
-                "totalRangeMax",
+        valuePositionBinding: {
+            title: "Value position binding",
+            oneOf: [
+                {
+                    type: "number",
+                },
+                {
+                    type: "string",
+                    enum: [
+                        "selectedRangeMin",
+                        "selectedRangeMax",
+                        "totalRangeMin",
+                        "totalRangeMax",
```

Comment:
Looks like you are missing a few other potential values:
- "constrainedRangeMin"
- "constrainedRangeMax"

Reply:
Good catch, missing in base component too.